lib/Drush/CommandFiles/core/DrupliconCommands.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Drush\CommandFiles\core;
+
+class DrupliconCommands {
+
+  /*
+   * @todo not working until we modernize drush_get_global_options().
+   * @hook alter help
+   */
+  public function help($command) {
```

Comment:
@hook alter isn't called with a `$command` array; it is called with the same parameters as the main command method, which is to say a commandname string.